### ISSUE TITLE
OpenZWave with Razberry GPIO Board

### ISSUE THREAD

**Issue by nbibin001** [OpenZWave/node-openzwave-shared#217]
Hi i am trying to get OpenZWave to work with the Razberry GPIO board with Raspberry Pi 3. So I compiled and installed OpenZWave as suggested by the the raspberry pi installation guide. But when I try to run MinOZW I get an error as below. Could you please help me figure if I am doing something wrong ? Really appreciate your help.
![image](https://user-images.githubusercontent.com/15659590/36374887-f7b535e8-153a-11e8-8e8d-27d7ed7f4279.png)

![image](https://user-images.githubusercontent.com/15659590/36374797-a7a74550-153a-11e8-9726-bd334c845b3a.png)

**Comment by nbibin001** [comment]
I think I have figured this one out. The problem was the OpenZWave by default is expected to work with USB. So I changed it from USB (ttyUSB0) to ttyAMA0 and it started working. I was able to run the test.js example script provided, it scanned and listed the Razberry Zwave controller attached to the PI. 

I think now I need some help for adding a new device. I have a GE Plug in Smart ZWave dimmer. I am trying to add this device to the network. Is there any sample code available to add a node ? Appreciate any help :) 

![image](https://user-images.githubusercontent.com/15659590/36379609-49a9a1b0-154d-11e8-9074-6180dc46f07b.png)
